src/ducks/lagret-filter-ui-state.ts

```diff
@@ -0,0 +1,111 @@
+import {OrNothing} from "../utils/types/types";
+import {LagretFilter} from "./lagretFilter";
+
+// Actions
+export const VELG_LAGRET_FILTER = 'lagretfilter_velg/VELG_LAGRET_FILTER';
```

Comment:
aren't these for mine filter? since the actions for veiledergruppe is from line 10?

Reply:
`export const VELG_LAGRET_FILTER = 'lagretfilter_velg/VELG_MINE_FILTER';`

Reply:
aha...I will fix. I am bit confused when to use mine filter and when lagret filter